src/Files.App/UserControls/OngoingTasksFlyout.xaml

```diff
@@ -67,7 +67,7 @@
 
 							<ProgressBar
 								x:Name="ProgressBar"
-								Margin="0,0,12,12"
+								Margin="0,-24,12,12"
```

Comment:
What's the negative margin for?

Reply:
The problem occurs when the title and the message are displayed in one line. So I added a line break to the message to force multiple lines. This negative margin is to layout the progress bar ignoring the line break.